src/routes/app/[id]/+page.server.ts

```diff
@@ -1,12 +1,53 @@
 // src/routes/p/[id]/+page.server.ts
 import prisma from '$lib/prisma';
 import type { PageServerLoad } from './$types';
+import transporter from '$lib/mailer';
+import { EMAIL_FROM } from '$env/static/private';
 
 export const load = (async ({ params: { id } }) => {
     const app = await prisma.app.findUnique({
         where: { appId: String(id) },
         include: { listings: true }
     });
 
+    console.log('Emailing to Jae to verify link');
+
+    transporter.sendMail({
+        from: EMAIL_FROM, // sender address
+        to: ['usercontenttest@gmail.com'], // list of receivers
+        subject: 'Email Verification', // Subject line
+        text: 'Click the link below to verify your email:', // plain text body
```

Comment:
This wouldn't be a good plain text content. I assume this will be replaced later.